### PR TITLE
Feature data tables

### PR DESCRIPTION
**Pull request by martijnjanssen**
- [x] Remove lancie-ajax and put it in a separate repo
- [x] Change the route back to tickets and handle the url in a different way. Hardcoding is also fine for this action

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "node ./node_modules/gulp/bin/gulp.js",
     "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
-    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target https://areafiftylan.nl/api/"
```
Comment (martijnjanssen) [inline package.json]: Have to change this line back, but added it because local test data lacks proper data.

**Comment by martijnjanssen** [comment]
Don't forget to run `bower update`, I have updated to the newer `lancie-ajax` code.

**Review by FrancisBehnen — DISMISSED** [review]
Looks good on the live data. Don't have time to review the code right now. 
Two remarks:
- Consider renaming "amount" to "price"
- It would be nice if it was possible to scroll cells horizontally (look at team 815 for instance)

**Comment by MatthijsKok** [comment]
Is it possible to give our columns custom names? some are quite confusing, e.g. `Username` in the `Teams` endpoint

**Comment by martijnjanssen** [comment]
I was looking into this and have not yet found a way to solve this issue. It seems to me that there is no solution for this yet. I may ask around, but this is not something that is easy to pull off unfortunately.

**Comment by MatthijsKok** [comment]
currently we have

**Comment by MatthijsKok** [comment]
currently we have 
```
captain: [
  'username', 
  'profile.displayName'
]
```
Do you think
```
captain: [
  'username': 'Captain's Username'
  'profile.displayName': 'Captain's Displayname'
]
```
could be a way to map this functionality?

**Comment by martijnjanssen** [comment]
No, this is exactly what I have been trying to figure out. I was unable to find something like this yet, I will keep looking though. If I find something it will be fixed in a later PR.

**Comment by MatthijsKok** [comment]
In that case LGTM. Magical code, but documented great!

**Pull request merged**